tools/slang-unit-test/unit-test-cooperative-type-metadata.cpp

```
@@ -0,0 +1,420 @@
+[shader("compute")]
+[numthreads(1, 1, 1)]
+void computeMain()
+{
+    CoopVec<half, 4> vec;
+    for (int i = 0; i < vec.getCount(); ++i)
+        vec[i] = half(i + 1);
+    coopVecReduceSumAccumulate(vec, output, 0);
+}
+)";
+
+SLANG_UNIT_TEST(cooperativeMetadata_emptyShader)
```

Cppcheck (2.20.0)

[style] The function 'DoubleAsInt64' is never used.

(unusedFunction)